src/test/java/org/springframework/integration/dsl/test/jms/JmsTests.java

```diff
@@ -221,11 +249,38 @@ public IntegrationFlow jmsInboundFlow() {
 		}
 
 		@Bean
+		public IntegrationFlow pubSubFlow() {
+			return IntegrationFlows.from(Jms.publishSubscribeChannel(this.jmsConnectionFactory)
+											.destination("pubsub"))
+									.bridge(e -> e.id("bridgePubSub"))
+									.channel(jmsPubSubBridgeChannel())
+									.get();
+		}
+
+		@Bean
+		public PollableChannel jmsPubSubBridgeChannel() {
+			return new QueueChannel();
+		}
+
+		@Bean
 		public IntegrationFlow jmsMessageDriverFlow() {
 			return IntegrationFlows
 					.from(Jms.messageDriverChannelAdapter(this.jmsConnectionFactory)
 							.destination("jmsMessageDriver"))
 					.<String, String>transform(String::toLowerCase)
+					.channel(jmsOutboundInboundReplyChannel())
+					.get();
+		}
+
+		@Bean
+		public IntegrationFlow jmsMessageDrivenFlowWithContainer() {
+			return IntegrationFlows
+					.from(Jms.messageDriverChannelAdapter(
+								Jms.container(this.jmsConnectionFactory, "containerSpecDestination")
+									.pubSubDomain(false)
+									.get())
```

Comment:
Isn't so important, but looks like we have in other places an overloaded version for similar factory method which accepts just `Spec` to avoid this extra `.get()`.
Of, course we can forget that right now and do that on demand.